mapie/metrics.py

```diff
@@ -307,3 +311,407 @@ def top_label_ece(
         )
     ece /= len(labels)
     return ece
+
+
+def regression_coverage_score_v2(
```

Comment:
add a test to check that for the same value of alpha, regression_coverage_score_v2 and regression_coverage_score return the same value